workloads/fio/fio.py

```diff
@@ -0,0 +1,80 @@
+import logging
+
+from ocs import exceptions
+
+log = logging.getLogger(__name__)
+
+DISTROS = {"Debian": "apt-get", "RHEL": "yum"}
+
+
+def find_distro(io_pod):
+    """
+    Find whats the os distro on pod
+
+    Args:
+        io_pod (Pod): app pod object
+
+    Returns:
+        distro (str): representing 'Debian' or 'RHEL' as of now
+    """
+    for distro, pkg_mgr in DISTROS.items():
+        try:
+            io_pod.exec_cmd_on_pod(f"which {pkg_mgr}")
+        except exceptions.CommandFailed:
+            log.debug(f"Distro is not {distro}")
+        else:
+            return distro
+
+
+def setup(**kwargs):
+    """
+    setup fio workload
+
+    Args:
+        **kwargs (dict): fio setup configuration
```

Comment:
Is this documented somewhere?

Reply:
@zmc ,
This is just setup part for fio, as of now I am just doing 'yum install'.

Reply:
Sorry, I don't understand. I'm asking what key/value pairs `kwargs` might contain. So far I see a 'pod' key used. If this is a standard interface that's documented elsewhere, perhaps you could point me to that?

Reply:
In case of fio nothing is required except pod, may be I can document that. The reason I made it as **kwargs is because in future if we want to setup fio containerized image then we need whole resource yaml dict to be passed in here. I hope that is what you meant.

Reply:
Yeah, that's what I meant. I'd suggest either just taking the args you need (and documenting them), or documenting the structure of the dict you're taking here.

Reply:
Done, added description about possible **kwargs content.